_posts/2019/2019-07-31-consistency.md

```diff
@@ -11,93 +11,89 @@ authors: ["Christina"]
 By taking the suggestions from the translation memory into account, you can thus easily translate recurring terms and phrases consistently.
 
 ## Glossaries
 
-I think we all agree that translation memories are a great way to support consistency in translations.
+I think we all agree that a translation memory is a great tool to support consistency in translations.
 But what about consistency in copywriting?
 If there is no text that the translation memory can compare, how should it be able to make suggestions?
 
-So, copywriters have a green field and can come up with whatever they want?
-That's not true either.
-Of course, they also need to take care of using the common terms and phrases.
+Are copywriters free to use whatever terminology they want?
+Of course not.
+They also need to take care of using the common terms and phrases.
 But it's a bit harder to find them.
-Glossaries can help here.
-They cannot only be used to explain certain terms and concepts to externals like translation agencies.
-By [maintaining a glossary](https://developer.epages.com/blog/language-and-localization/five-compelling-reasons-why-you-should-care-about-terminology/){:target="_blank"} you also facilitate the life of your copywriters.
-If they are e.g. wondering if they should say *user name* or *username*, or *ecommerce* or *e-commerce*, they can check the spelling in the glossary.
-This way the most important terms of your product will be spelled consistently throughout your software.
+Glossaries do not only help to explain certain terms and concepts to external translators.
+[Maintaining a glossary](https://developer.epages.com/blog/language-and-localization/five-compelling-reasons-why-you-should-care-about-terminology/){:target="_blank"} also facilitates a copywriters's life.
+If they are e.g. wondering if to use *user name* or *username*, or *ecommerce* or *e-commerce*, they can check the spelling in the glossary.
+This way the most important terms of the product will be spelled consistently throughout the software.
 
 ## UI text guidelines
 
-Glossaries can support copywriters with the right spelling of a term.
-But what about whole sentences?
-Or the general grammar, and tone of voice?
+Glossaries can support copywriters with the right spelling of a term or a sentence.
+But what about the general grammar, and tone of voice?
 You'll need a guideline for these topics.
-The content of the guideline is totally up to you and your product.
-You could start by establishing a structure for recurring text types, and decide how to apply grammar rules to these types.
-This way you can e.g. make sure that all checkbox labels start with a verb.
+The content of the guideline is totally up to the company and the product.
+Establishing a structure for recurring text types, and deciding how to apply grammar rules to these types could be the first steps.
+This way it could e.g. be achieved that all checkbox labels start with a verb.
```

Comment:
Starting off with such a guideline helps, let's say to align that all...
That's already a good move into the right direction.